docs/nuxt.config.js

```diff
@@ -14,6 +14,10 @@ hljs.registerLanguage('bash', require('highlight.js/lib/languages/bash')) // inc
 hljs.registerLanguage('shell', require('highlight.js/lib/languages/shell'))
 hljs.registerLanguage('plaintext', require('highlight.js/lib/languages/plaintext'))
 
+// Constants which should be grabbed from package.json
+// Bootstrap minor version (for links)
+const bootstrapDocsVersion = '4.3'
```

Comment:
The more I think about it I don't like this change...
Maybe it's better to just run a search and replace on the `.md` files.

The other changes are good!

Reply:
Yeah, it might be easy for future devs to forget to update this variable.

Reply:
Removed that part of the PR